test/specs/elements/Header/Header-test.js

```diff
@@ -65,4 +67,28 @@ describe('Header', () => {
       wrapper.should.not.have.descendants('HeaderContent')
     })
   })
+
+  describe('subheader', () => {
+    it('adds HeaderSubheader as child', () => {
+      const text = faker.hacker.phrase()
+
+      shallow(<Header subheader={text} />)
+        .find('HeaderSubheader')
+        .should.have.prop('content', text)
+    })
+    it('adds HeaderSubheader as child when given a name to icon prop', () => {
+      const text = faker.hacker.phrase()
+
+      shallow(<Header icon='user' subheader={text} />)
+        .find('HeaderSubheader')
+        .should.have.prop('content', text)
```

Comment:
Thanks for shallow rendering and only testing this prop was passed, instead of testing the Subheader's handling of the prop in the Header test 👍  This is a great test.